src/NBInclude.jl

```diff
@@ -57,7 +57,10 @@ it is assigned a number `+N` for the `N`-th nonempty cell.  If `renumber`
 is set to `true`, then the cell numbers saved in the notebook are ignored
 and each cell is assigned a consecutive number `N`.
 """
-function nbinclude(path::AbstractString; renumber::Bool=false)
+function nbinclude(path::AbstractString; renumber::Bool=false,
+                                         counters = nothing,
```

Comment:
You could just default to `typemin(Int):typemax(Int)` (and `regexp::Regex=r""`)and then you could just do `counter ∉ counters` without checking for `nothing`.